src/main/io/ledstrip.c

```diff
@@ -948,10 +946,10 @@ void ledStripUpdate(uint32_t currentTime)
         // sanitize timer value, so that it can be safely incremented. Handles inital timerVal value.
         // max delay is limited to 5s
         int32_t delta = cmp32(now, timerVal[timId]);
-        if (delta < 0 && delta > -LED_STRIP_MS(5000))
+        if (delta < 0 && delta > -HZ_TO_MICROS(0.2f))
```

Comment:
Yeah I guess I should have gone for 5000000 there. What I wanted to achieve is to reduce the number of macros that are used for the same thing.